src/Microsoft.ML.FastTree/FastTreeArguments.cs

```diff
@@ -594,7 +643,7 @@ public abstract class BoostedTreeOptions : TreeOptions
         public bool BestStepRankingRegressionTrees = false;
 
         /// <summary>
-        /// Should we use line search for a step size.
+        /// Determines whether to we use line search for a step size.
```

Comment:
>to we use [](start = 31, length = 9)

`to we use` -> `to use`

Reply:
fixed

---
In reply to: [265814516](https://github.com/dotnet/machinelearning/pull/2970#discussion_r265814516) [](ancestors = 265814516)